cinch/site.yml

```diff
@@ -29,6 +29,7 @@
   hosts: jenkins_master
   roles:
     - jenkins_master
+    - ntp
```

Comment:
Would there be any downside to having Jenkins slaves also include the NTP role?  I know that Jenkins can get upset if slaves have clocks that are out of sync with the master.